compiler/src/iree/compiler/Codegen/LLVMCPU/LLVMCPUSelectLoweringStrategy.cpp

```diff
@@ -67,6 +67,9 @@ verifyLoweringConfiguration(FunctionOpInterface funcOp,
                             IREE::Codegen::TranslationInfoAttr translationInfo,
                             F verificationFn) {
   auto walkResult = funcOp.walk([&](Operation *op) -> WalkResult {
+    if (isa<IREE::LinalgExt::CustomOp>(op)) {
+      return WalkResult::advance();
+    }
```

Comment:
Can we just change the verification function to handle these ops?

Reply:
no that wont work. The goal is that ops within the body are what truly hold the lowering config. I dont think that works.

Reply:
by handle I just meant `return WalkResult::advance()` in the verifier rather than here. Maybe I'm misreading the code but doesn't the `verificationFn` return a walk result?

Reply:
verification returns a `LogicalResult`. If I advance it, it will go to the actual operation that is carrying the lowering config and then verifies.